### DISCUSSION TITLE
Is there any translation option for ... ?

### DISCUSSION THREAD

**Discussion by ghost** [saadeghi/daisyui#1025] [Q&A] [answered]
No option to translate this sentense in json file

`Once you installed daisyui, you can use component classes like btn, card etc.`

It's same in all langs

Link: https://daisyui.com/docs/use/

Let me know of any thoughts about it

**Comment by ghost** [comment]
Same is 
If you use daisyUI `prefix` option (like daisy-) and Tailwind CSS prefix option (like tw-) together, classnames will be prefixed like this: tw-daisy-btn.

Link: https://daisyui.com/docs/config/

**Comment by saadeghi** [comment, marked as answer]
Fixed! thank you for reporting.
